addons/binding/org.openhab.binding.wifiled/src/main/java/org/openhab/binding/wifiled/handler/LEDStateDTO.java

```diff
@@ -149,9 +154,9 @@ public LEDStateDTO withIncrementedProgramSpeed(int step) {
     }
 
     public Color getColor() {
-        float hue = (float) this.getHue().intValue() / 360;
-        float saturation = (float) this.getSaturation().doubleValue() / 100;
-        float brightness = (float) this.getBrightness().intValue() / 100;
+        float hue = (float) this.getHue().doubleValue() / 360.0f;
```

Comment:
Old as in existing :-)

> I think this is a bug but would like to get educated of the reason why casting to int is done here.

Casting is maybe not correct, rounding should be better. But as mentioned before storing it as integer values feels like I would expect.